docker-compose.yml

```
@@ -5,18 +5,17 @@ services:
     build:
       context: .
       dockerfile: ./frontend/Dockerfile
-    container_name: vibes-yay
+    container_name: vibes-frontend-docker
```

```suggestion
    container_name: vibes-frontend
```
"docker" is somewhat redundant. not a big issue though